src/injected/main-window-initializer.ts

```diff
@@ -328,11 +329,26 @@ export class MainWindowInitializer extends WindowInitializer {
             tabStopRequirementActionMessageCreator,
         );
 
-        const analyzerProvider = new AnalyzerProvider(
-            this.manualTabStopListener,
+        const tabStopsRequirementResultProcessor = new TabStopsRequirementResultProcessor(
+            this.featureFlagStoreProxy,
             this.tabStopRequirementRunner,
             tabStopRequirementActionMessageCreator,
+            this.visualizationScanResultStoreProxy,
+        );
+
+        const tabStopsRequirementResultProcessorForFocusTracking =
+            new TabStopsRequirementResultProcessor(
+                this.featureFlagStoreProxy,
+                null,
+                tabStopRequirementActionMessageCreator,
+                this.visualizationScanResultStoreProxy,
+            );
```

Comment:
At the moment, can't think of a way to initialize just one TabStopsRequirementResultProcessor with a tabStopRequirementRunner or null to pass into AnalyzerProvider.

Reply:
Maybe we can always have tabStopRequirementRunner be non-null, but pass something in start() to differentiate between the case when we want to use it and when we don't?